galaxy_importer/collection.py

```diff
@@ -58,7 +58,10 @@ def import_collection(file, filename=None, logger=None, cfg=None):
 
 
 def _import_collection(file, filename, logger, cfg):
-    with tempfile.TemporaryDirectory() as tmp_dir:
+    tmp_root_dir = None
+    if cfg:
+        tmp_root_dir = cfg.tmp_root_dir
```

Comment:
looking again at this, since we have a default config, `if cfg` will always resolve to true, recommend:
`tmp_root_dir = cfg.tmp_root_dir`